### PR TITLE
Added starter workflow and updated Readme

### PR DESCRIPTION
**Pull request by lmrb-1968**
Added starter workflow and updated Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,17 +90,25 @@ jobs:
...
+        # uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
+        uses: actions/checkout@v3.5.3
+        with:
+          # The default depth of 1 commit is not enough for some scans 
+          fetch-depth: 0
         
       - name: Xygeni-Scanner
-        uses: xygeni/xygeni-action@v2.0
+        # Pinned: xygeni/xygeni@4e94e5ea737f5ebfedac8b1a5b75d8c60c21932d
+        uses: xygeni/xygeni-action@v2.1
```
Comment (Garcel) [inline README.md]: Probably v.3.2.0 should be used here. (SHA 4e94e5ea737f5ebfedac8b1a5b75d8c60c21932d).

https://github.com/xygeni/xygeni-action/releases/tag/v3.2.0

File: workflow-templates/code-scanning/xygeni.yml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        # uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
+        uses: actions/checkout@v3.5.3
+        with:
+          # Full git history for better code tampering detection.
+          fetch-depth: 0
+
+      - name: Xygeni scan
+        # You may pin an action SHA, like this:
+        # uses: xygeni/xygeni@4e94e5ea737f5ebfedac8b1a5b75d8c60c21932d
+        uses: xygeni/xygeni@2.1
```
Comment (Garcel) [inline workflow-templates/code-scanning/xygeni.yml]: Probably v.3.2.0 should be used here. (SHA 4e94e5ea737f5ebfedac8b1a5b75d8c60c21932d).

https://github.com/xygeni/xygeni-action/releases/tag/v3.2.0

**Review by Garcel — CHANGES_REQUESTED** [review]
Probably v.3.2.0 should be included in the documentation/ templates.

File: workflow-templates/code-scanning/xygeni.yml (hunk 2 above)
Comment (lmrb-1968) [reply workflow-templates/code-scanning/xygeni.yml]: Done. Added @v3.2.0

File: README.md (hunk 1 above)
Comment (lmrb-1968) [reply README.md]: Added

**Review by lmrb-1968 — COMMENTED** [review]
Added v3.2.0 instead of the v2.1. I was waiting for the release to write the definite version name.

**Review by Garcel — APPROVED** [review]

**Pull request merged**